app/src/layouts/kanban/kanban.vue

```diff
@@ -88,6 +88,10 @@ function saveChanges() {
 	editTitle.value = '';
 }
 
+const titleFieldConfiguration = computed<Field | undefined>(() => {
+	return props.fieldsInCollection.find((field) => field.field === props.layoutOptions?.titleField);
+});
+
 const textFieldConfiguration = computed<Field | undefined>(() => {
 	return props.fieldsInCollection.find((field) => field.field === props.layoutOptions?.textField);
 });
```

Comment:
@formfcw thoughts on maybe combining these into a single computed prop, seeing as they're basically exactly the same?

```
const fieldConfigurations = computed<{ textField: Field | undefined; titleField: Field | undefined }>(() => {
	return {
		textField: props.fieldsInCollection.find((field) => field.field === props.layoutOptions?.textField),
		titleField: props.fieldsInCollection.find((field) => field.field === props.layoutOptions?.titleField),
	};
});
```